src/org/labkey/test/tests/ExpTest.java

```diff
@@ -175,7 +176,7 @@ public void testSteps()
         _customizeViewsHelper.addColumn("WrappedRowId/Created");
         _customizeViewsHelper.applyCustomView();
         // Verify that it was joined and formatted correctly
-        textCount = countText(dateFormat.format(new Date()));
+        textCount = countText(dateFormat.format(importDate));
         // records for generated files experiment.xar.log and experiment.xar.xml may have been created automatically
         assertTrue("Number of records is not as expected", textCount == 5 || textCount == 7);
```

Comment:
It might be helpful to future us to log the value of `dateFormat.format(importDate)` when this fails; that plus the evidence in the screencap will make it obvious what happened